tasks/set_boot_order_director.yml

```diff
@@ -1,70 +1,35 @@
 ---
 - name: Clear redfish job queues
-  shell: |
-    source {{ badfish_venv }}/bin/activate;
-    python3 badfish.py -H mgmt-{{ item[1].stdout }}  -u quads -p {{ chassis_password }} --clear-jobs --force
-  args:
-    chdir: "{{ ansible_user_dir }}/badfish/src/badfish"
-  when: vendors is defined and item[0] == "dell"
-  with_together:
-    - "{{ vendors }}"
-    - "{{ host_list.results }}"
+  become: yes
+  shell: podman run -it --rm quay.io/quads/badfish --host-list /bad-hosts -u quads -p {{ chassis_password }} --clear-jobs --force
```

Comment:
It would be better if you create separate PR's for badfish container and parallel execution to set boot order for readability